lib/counter.rb

```diff
@@ -9,7 +11,24 @@ def increment
     @count += 1
   end
 
+ def decrement
+   @count -= 1
+ end
+
+ def reset
+   @count = 0
+ end
+
+ def time
+   Time.now.round
```

Comment:
Does this implementation really fulfill the requirement of returning the time **of the last increment/decrement action**?